Microsoft.Recognizers.Text.DateTime/English/Extractors/EnglishDateExtractorConfiguration.cs

```diff
@@ -60,7 +60,7 @@ public class EnglishDateExtractorConfiguration : IDateExtractorConfiguration
 
         public static readonly Regex StrictWeekDay =
             new Regex(
-                DateTimeDefinitions.StrictWeekDay,
+                DateTimeDefinitions.WeekDayRegex,
                 RegexOptions.IgnoreCase | RegexOptions.Singleline);
```

Comment:
It is wield that the name of the parameter is StrictWeekDay but uses the DatetimeDefinitions.WeekDayRegex. What is the difference between the two regexes?